plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryTypeUtils.java

```diff
@@ -66,7 +68,10 @@ public static Object readNativeValue(Type type, Block block, int position)
             return null;
         }
 
-        // TODO https://github.com/trinodb/trino/issues/13741 Add support for decimal, time, timestamp with time zone, geography, map type
+        // TODO https://github.com/trinodb/trino/issues/13741 Add support for time, timestamp with time zone, geography, map type
+        if (type instanceof DecimalType) {
```

Comment:
nit: Move to after `DOUBLE ` for grouping numeric types together.

Reply:
Moved